cs/commandstation/ProgrammingTrackFrontend.hxx

```diff
@@ -216,6 +223,8 @@ class ProgrammingTrackFrontend
     ERROR_UNIMPLEMENTED_CMD = openlcb::Defs::ERROR_UNIMPLEMENTED_CMD,
     /// Error code when the request arguments are invalid.
     ERROR_INVALID_ARGS = openlcb::Defs::ERROR_INVALID_ARGS,
+    /// Error returned when the programming track is disabled by config.
+    ERROR_PGMTRACK_DISABLED = openlcb::Defs::ERROR_SRC_NOT_PERMITTED | 1,
```

Comment:
What are these "magic" numbers that are getting or'd in?

Reply:
These are the lowest nibble of the error code. The OpenLCB TN says that those values are defined by the device manufacturer.